src/main/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepExecution.java

```diff
@@ -408,19 +413,13 @@ public static final class PlaceholderTask implements ContinuedTask, Serializable
         private final String runId;
         /**
          * Unique cookie set once the task starts.
-         * Serves multiple purposes:
-         * identifies whether we have already invoked the body (since this can be rerun after restart);
```

Comment:
I guess it is still being used in this way (`hasStarted` / `willContinue`) but best leave that detail to those methods.